doc/api/test.md

```diff
@@ -635,6 +647,10 @@ The following built-reporters are supported:
 * `junit`
   The junit reporter outputs test results in a jUnit XML format
 
+* `lcov`
```

Comment:
Can you also add the `lcov` reporter to the import & require examples from `node:test/reporters` at lines 665 and 669?

Reply:
# > - - [ ] []()__